Puzzlepart/did
# Timesheet statusbar/mobile improvements

### Your checklist for this pull request

### Review checklist

### Description
Timesheet statusbar/mobile improvements

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Make sure you get a *.loca.lt subdomain, this is needed for testing on mobile
3. Open your dev-domain on your phone
4. Navigate to an unconfirmed timesheet, preferrably one you've forecasted but not confirmed
5. Observe that each StatusBar element is on its own line


![image](https://user-images.githubusercontent.com/7300548/98870430-94219580-2473-11eb-9766-b579468f57c0.png)
